src/main/java/org/prebid/server/bidder/model/BidderBid.java

```diff
@@ -23,6 +24,11 @@
      */
     BidType type;
 
+    /**
+     * This will become response.seatbid[i].bid.ext.prebid.type" in the final OpenRTB response
+     */
+    BidVideo bidVideo;
```

Comment:
i think this can be redundant. According to Go implementation `bidVideo` is used for applying of `category mapping` which is not supported on our side yet. Moreover, we don't need to add video specific object to `BidderBid` since it is used by all kind of auctions.

So, when we'll have ability to add `category mapping` feature - we can just add common for all bidders object to `bid.ext` for video processing. Just a thought.

As a result, let's `video` endpoint uses general auction flow but has specific incoming request.